apps/vmq_diversity/priv/vmq_diversity.schema

```diff
@@ -20,11 +20,31 @@
 %% order defined by the names given, i.e., the script with <name>
 %% 'script1' is started before the plugin with <name> 'script2'.
 %%
+%%
 %% Scripts loaded like this are loaded after the scripts in the
 %% default script dir.
+%%
+%% 
+%% Several default scripts are bundled with VerneMQ to authenticate
+%% against Postgres, MySQL, MongoDB, and Redis. Adapt the database configuration
+%% at the top of the script to match your setup.
+%%
+%%
+%% For PostgreSQL:
+%%   vmq_diversity.postgres.file = {{luascripts_dir}}/auth/postgres.lua
+%%
+%% For MySQL:
+%%   vmq_diversity.mysql.file = {{luascripts_dir}}/auth/mysql.lua
+%%
+%% For MongoDB:
+%%   vmq_diversity.mongodb.file = {{luascripts_dir}}/auth/mongodb.lua
+%%
+%% For Redis:
+%%   vmq_diversity.redis.file = {{luascripts_dir}}/auth/redis.lua
```

Comment:
The `luascripts_dir` will by default point to `share/lua` and the scripts you mention won't be there I believe. So maybe they should be copied there in the vernemq.conf. Is it confusing that we say that anything in `luascripts_dir` will be automatically loaded and then have specific, non-autoload files in the `luascripts_dir/auth` folder? Maybe we should change the structure to `luascripts_dir/autoload` and have the non-autoload scripts in the `luascripts_dir` folder?